playkit/src/main/java/com/kaltura/playkit/drm/WidevineClassicCompat.java

```diff
@@ -0,0 +1,34 @@
+package com.kaltura.playkit.drm;
+
+import android.annotation.TargetApi;
+import android.os.Build;
+import android.support.annotation.Nullable;
+
+import com.kaltura.playkit.PKLog;
+
+import java.io.FileDescriptor;
+import java.lang.reflect.Method;
+
+/**
+ * Created by gilad.nadav on 27/04/2017.
+ */
+
+public class WidevineClassicCompat {
```

Comment:
Add comment: 

> See description of WidevineClassicDrm.fdToString() for details on this class. This implementation was separated from WidevineClassicDrm to prevent a VerifyError on older devices.